parts/linux/cloud-init/artifacts/sync-tunnel-logs.sh

```diff
@@ -48,7 +48,7 @@ echo "Starting inotifywait..."
 # Monitor for changes
 inotifywait -q -m -r -e delete,create $SRC | while read DIRECTORY EVENT FILE; do
     case $FILE in
-        aks-link-*_kube-system_*.log | azure-cns-*_kube-system_*.log | cilium-*_kube-system_*.log | konnectivity-*_kube-system_*.log | tunnelfront-*_kube-system_*.log)
+        *_kube-system_*.log)
```

Comment:
do we actually need this case statement now since we want all files?

Reply:
ah nvm still catches other namespaces otherwise

Reply:
Right. It could be an if instead, but case has the nice wildcard matching built in, and it's easy to expand back out if we decide to add something like calico-system or tigera-operator in the future.